modules/javafx.controls/src/test/java/test/com/sun/javafx/scene/control/behavior/BehaviorCleanupTest.java

```diff
@@ -188,8 +377,46 @@ public void testListViewBehaviorDispose() {
 
   //------------------ setup/cleanup
 
+    /**
+     * Ensures the control is shown and focused in an active scenegraph.
+     *
+     * @param control the control to show
+     */
+    protected void showControl(Control control) {
+        showControl(control, true);
+    }
+
+    /**
+     * Ensures the control is shown in an active scenegraph. Requests
+     * focus on the control if focused == true.
+     *
+     * @param control the control to show
+     * @param focused if true, requests focus on the added control
+     */
+    protected void showControl(Control control, boolean focused) {
+        if (root == null) {
+            root = new VBox();
+            scene = new Scene(root);
+            stage = new Stage();
+            stage.setScene(scene);
+        }
+        if (!root.getChildren().contains(control)) {
+            root.getChildren().add(control);
```

Comment:
The controls added to root are not removed. I think we should clear the scenegraph after execution of each test.
suggesting to add following call in the cleanup method,
```
if (root != null) {
    root.getChildren().removeAll();
}
```

Reply:
Hmm ... don't quite understand: the cleanup follows the same pattern used across many controls/skin tests 

    @After
    public void cleanup() {
        if (stage != null) {
            stage.hide();
        }

The stage is created at most once per test method, and allows to add more controls in that same test method, it's hidden after running each test. Running the next text, there's no reference to the old .. why should we remove its children also? Or maybe I misunderstand what you are suggesting :)

Reply:
Oops, the current code is proper. I missed the cleanup somehow. This looks good.